massa-execution-worker/src/tests/scenarios_mandatories.rs

```diff
@@ -449,6 +449,192 @@ fn test_nested_call_gas_usage() {
     );
 }
 
+/// Test the recursion depth limit in nested calls using call SC operation
+///
+/// Create a smart contract and send it in the blockclique.
+/// This smart contract have his sources in the sources folder.
+/// It calls the test function that have a sub-call to the receive function and send it to the blockclique.
+/// We are checking that the gas is going down through the execution even in sub-calls.
+///
+/// This test can fail if the gas is going up in the execution
+#[test]
+fn test_nested_call_recursion_limit_reached() {
+    // setup the period duration
+    let exec_cfg = ExecutionConfig {
+        max_recursive_calls_depth: 0,
```

Comment:
Add a small comment here indicating that we expect the test to fail as soon as the SC did a first call/local_call?